packages/ui-library/src/components/virtualized-table/index.tsx

```diff
@@ -59,7 +59,13 @@ const StyledBodySmallLabelTypography = styled(Typography).attrs(
   })
 )``;
 
-interface BaseContext {}
+interface BaseContext {
```

Comment:
Could we change the name to something more descriptive? Like VirtualizedTableContext